src/Acf.php

```diff
@@ -185,4 +199,23 @@ private static function apply_default_transform( $field_obj ) {
 	private static function get_transform_class_name( $field_type ) {
 		return '\\' . __NAMESPACE__ . '\\Acf\\Transforms\\' . str_replace( '_', '', ucwords( $field_type, '_' ) );
 	}
+
+	/**
+	 * Get an ACF group
+	 *
+	 * @param $group_id
+	 * @return bool
+	 */
+	private static function get_group( $group_id ) {
+		// @codingStandardsIgnoreStart
+		// Ignore use WP_Query rule because we don't know if this will be a sub-query and hence create complications.
+		$groups = get_posts( [
```

Comment:
Is there a way to cache the values inside of an array for instance if we are looking for the gruoup `group_56f57c664a38e` from 10 childs we are going to make a request to this function 10 times, instead we should cache the values of each group in associative array like: 

```php
if ( !  isset( $groups['group_56f57c664a38e'] ) ){
  $groups['group_56f57c664a38e'] = // function logic here.
 }
return $groups['group_56f57c664a38e'];
```

With this approach we only call the function 1 no matter how many times we have to search for a group.